dallinger/command_line.py

```diff
@@ -836,13 +843,11 @@ def export(app, local):
 
     for table in all_tables:
         subprocess.check_call(
-            "psql -d dallinger --command=\"\\copy " + table + " to \'" +
+            "psql -d " + app_name(id) +
+            " --command=\"\\copy " + table + " to \'" +
             os.path.join(subdata_path, table) + ".csv\' csv header\"",
             shell=True)
```

Comment:
It'd be best to write this as follows to avoid potential for the shell to misinterpret things:
```python
subprocess.check_call([
    'psql', '-d', app_name(id),
    r'''--command="\copy {} to '{}.csv' csv header"'''.format(
        table, os.path.join(subdata_path, table)
    )
])
```